Libraries/ActionSheetIOS/RCTActionSheetManager.m

```diff
@@ -133,11 +133,17 @@ - (CGRect)sourceRectInView:(UIView *)sourceView
   if (message) {
     [items addObject:message];
   }
-  NSURL *URL = [RCTConvert NSURL:options[@"url"]];
-  if (URL) {
-    if ([URL.scheme.lowercaseString isEqualToString:@"data"]) {
+  NSArray<NSURL *> *URLS = [RCTConvert NSURLArray:options[@"url"]];
```

Comment:
This is a breaking API change that introduces crashes into all existing apps using it. Highly suggest introducing a "urls" argument instead.